snowflake/cmd/onboard.go

```diff
@@ -0,0 +1,128 @@
+		stackName, _ := cmd.Flags().GetString("stack-name")
+		bucketName, _ := cmd.Flags().GetString("bucket-name")
+		bucketPrefix, _ := cmd.Flags().GetString("bucket-prefix")
+		bucketRegion, _ := cmd.Flags().GetString("bucket-region")
+		keyID, _ := cmd.Flags().GetString("key-id")
+		keyRegion, _ := cmd.Flags().GetString("key-region")
+		warehouseName, _ := cmd.Flags().GetString("warehouse-name")
+		workdir, _ := cmd.Flags().GetString("workdir")
+		verbose := verbosity >= 2
+		ctx, cancel := context.WithTimeout(context.Background(), 300*time.Second)
+		defer cancel()
+		if bucketRegion == "" {
+			var err error
+			bucketRegion, err = GetBucketRegion(ctx, bucketName, region)
+			if err != nil {
+				return err
+			}
+		}
+		stateBackendURL := infra.S3StateBackendURL(bucketName, bucketPrefix, bucketRegion)
+		var secretsProviderURL string
+		if keyID != "" {
+			if keyRegion == "" {
+				keyRegion = region
+			}
+			secretsProviderURL = infra.KmsSecretsProviderURL(keyID, keyRegion)
+		}
+		mgr := infra.NewManager(logger, stackName, stateBackendURL, secretsProviderURL, region, warehouseName, workdir, verbose)
+		result, err := mgr.Onboard(ctx)
+		if err != nil {
+			return err
+		}
+		showResults(result)
+		fmt.Println("SUCCESS!")
+		fmt.Println("To update infrastructure, run 'theia-sf onboard' again")
+		fmt.Println("To destroy all infrastructure, run 'theia-sf offboard'")
+		return nil
+	},
+}
+
+func showResults(result *infra.Result) {
+	table := tablewriter.NewWriter(os.Stdout)
+	data := [][]string{
+		[]string{"Region", result.Region},
+		[]string{"Bucket Name", result.BucketName},
+		[]string{"Bucket Flows Folder", result.BucketFlowsFolder},
+		[]string{"Snowflake Database Name", result.DatabaseName},
+		[]string{"Snowflake Schema Name", result.SchemaName},
+		[]string{"Snowflake Flows Table Name", result.FlowsTableName},
+		[]string{"SNS Topic ARN", result.SNSTopicARN},
+		[]string{"SQS Queue ARN", result.SQSQueueARN},
+	}
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.AppendBulk(data)
+	table.Render()
+}
+
+func init() {
+	rootCmd.AddCommand(onboardCmd)
+
+	onboardCmd.Flags().String("region", GetEnv("AWS_REGION", defaultRegion), "region where AWS resources will be provisioned")
+	onboardCmd.Flags().String("stack-name", "default", "name of the infrastructure stack: useful to deploy multiple instances in the same Snowflake account or with the same bucket (e.g., one for dev and one for prod)")
+	onboardCmd.Flags().String("bucket-name", "", "bucket to store infra state")
+	onboardCmd.MarkFlagRequired("bucket-name")
+	onboardCmd.Flags().String("bucket-prefix", "antrea-flows-infra", "prefix to use to store infra state")
+	onboardCmd.Flags().String("bucket-region", "", "region where infra bucket is defined; if omitted, we will try to get the region from AWS")
+	onboardCmd.Flags().String("key-id", GetEnv("THEIA_SF_KMS_KEY_ID", ""), "Kms key ID")
+	onboardCmd.Flags().String("key-region", "", "Kms Key region")
```

Comment:
```suggestion
	onboardCmd.Flags().String("key-region", "", "Kms key region")
```
to be consistent with elsewhere